backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -883,6 +886,7 @@ describe('UserResolver', () => {
 
     describe('wrong optin code', () => {
       it('throws an error', async () => {
+        jest.clearAllMocks()
```

Comment:
What's the reason to clearAllMocks here?
There is no logger.test after the testcase.